backend/core/helpers.py

```
@@ -865,7 +878,7 @@
     return {"data": data, "names": names, "uuids": uuids}
 
 
-def csf_functions(user):
+def csf_functions(user, folder_id=None):
     (object_ids, _, _) = RoleAssignment.get_accessible_object_ids(
         Folder.get_root_folder(), user, AppliedControl
```

Ruff (0.8.2)

`AppliedControl` may be undefined, or defined from star imports

(F405)